sql/table.h

```diff
@@ -912,7 +912,7 @@ struct TABLE_SHARE {
     without explicit pk and corresponding warning was issued to disable
     repeated warning.
   */
-  bool rfr_lookup_warning;
+  bool rfr_lookup_warning{false};
```

Comment:
Does this point to a bug in lower versions?

Reply:
There is no issue with 5.7 because of `memset` in the constructor:
``` 
TABLE_SHARE() { memset(this, 0, sizeof(*this)); }
```